tests/test_astrom.py

```diff
@@ -55,15 +55,7 @@ def test_astrom():
 
     # check orientation is correct within 0.05 [deg]
     # and plate scale is correct within 0.5 [mas] (arbitrary)
-    test_result_platescale_x = astrom_cal.platescale[0] == pytest.approx(expected_platescale, abs=atol_platescale)
-    assert test_result_platescale_x
-    test_result_platescale_y = astrom_cal.platescale[1] == pytest.approx(expected_platescale, abs=atol_platescale)
-    assert test_result_platescale_y
-
-    test_result_platescale = test_result_platescale_x and test_result_platescale_y
-    print(f'\nPlate scale estimates from boresight_calibration() are accurate: {expected_platescale} +/- {atol_platescale}: ', end='')
```

Comment:
did you mean to delete these print statements and print_pass/print_fail below? I think these are needed for VAP testing.

Reply:
you probably need to modify this to handle the assert below, and print pass/fail based on the new assert.